pages/common/cb.md

```diff
@@ -0,0 +1,36 @@
+# cb
+
+> Cut, copy, and paste anything in the terminal.
+> More information: <https://github.com/Slackadays/Clipboard>.
+
+- Show all clipboards:
+
+`cb`
+
+- Copy a file to the clipboard:
+
+`cb copy {{path/to/file}}`
+
+- Copy some text to the clipboard:
+
+`cb copy "{{Some example text}}"`
+
+- Copy piped clipboard content:
+
+`echo "{{Some example text}}" | cb`
+
+- Paste clipboard content:
+
+`cb paste`
+
+- Pipe out clipboard content:
+
+`cb | cat`
```

Comment:
```suggestion
- Paste clipboard content (stored in `rawdata.clipboard` file by default):

`cb paste`

- Pipe out clipboard content to a text file:

`cb | cat - {{path/to/output_file.txt}}`
```
How about this?

Reply:
This will be false starting in 0.8.1 where the pasted file depends on what clipboard, history entry, and content type you're pasting, so we can't say it'll be any file in particular.

Reply:
Thanks for the clarification.